content/kots-cli/remove.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2020-11-20
+linktitle: "remove"
+title: kots remove
+weight: 90120
+---
+
+Remove application reference from Admin Console.
+Multiple applications can be installed using the `Add a new application` button in Admin Console or using the `kots install` command.
+All installed applications will appear in Admin Console.
+The `kots remove` command can be used to remove them.
+Note that deployed application will not be removed from the cluster, only it's reference will be removed from Admin Console.
+
+### Usage
+```bash
+kubectl kots remove [app-slug]
+```
+* _Replace `[app-slug]` with the slug for the KOTS application that should be removed (required)._
```

Comment:
You usually don't need to say "replace". People know how CLI docs work, right? Just say "app-slug is..."